src/views/register/register.tsx

```diff
@@ -0,0 +1,160 @@
+                        <div className="sec-box bg-white rounded px-10 mt-8 py-5" style={{ height: '645px', width: '464px' }}>
+                            <p className="text-black text-xl font-weight-bold mt-4 text-center" style={{ fontSize: '1.5rem' }}>
+                                Register
+                            </p>
+                            <button
+                                className="rounded border py-2 mt-6 w-full gap-3"
+                                style={{ display: 'flex', flexDirection: 'row', alignItems: 'center', justifyContent: 'center' }}
+                            >
+                                <Image src={google} alt="logo" />
+                                <p className="text-black font-weight-bold">LOGIN WITH GOOGLE</p>
+                            </button>
+                            <button
+                                className="rounded border py-2 text-black font-weight-bold mt-2 w-full gap-3"
+                                style={{ display: 'flex', flexDirection: 'row', alignItems: 'center', justifyContent: 'center' }}
+                            >
+                                <Image src={github} alt="logo" />
+                                <p className="text-black font-weight-bold">LOGIN WITH GITHUB</p>
+                            </button>
+                            <p className="text-black text-md font-weight-bold mt-5 text-center">or</p>
+                            <input
+                                type="text"
+                                className="form-control text-black bottom-border w-full mt-9"
+                                placeholder="Email"
+                                id="validationCustom01"
+                                required
+                            />
+                            <input
+                                type="text"
+                                className="form-control text-black bottom-border w-full mt-9"
+                                placeholder="Password"
+                                id="validationCustom01"
+                                required
+                            />
+                            <input
+                                type="text"
+                                className="form-control text-black bottom-border w-full mt-9"
+                                placeholder="Password repeat"
+                                id="validationCustom01"
+                                required
+                            />
+                            <button className="text-white font-weight-bold text-center bg-dark-600 w-full rounded py-2 mt-9">
+                                REGISTER
+                            </button>
+                            <p className="text-black text-md font-weight-bold mt-9 text-center">
+                                By clicking “Create account” or “Continue with Google” or “Continue with Github”, you agree to the
```

Comment:
Please look into the spaces inside this string.

Also add the following hyper-links 
TOS - https://www.notion.so/adityaagarwal/Terms-of-Use-0012b80699cc4b948cdae9e42983035b
Privacy policy - https://adityaagarwal.notion.site/Privacy-Policy-5f05315af636474797f1255d338a0d76?pvs=4